### PR TITLE
Production Release

### PR DESCRIPTION
**Pull request by reyrodrigues**
With changes for the week

### CODE REVIEW

File: simple_ui/static/scss/content.css [hunk 1]
```diff
@@ -67,28 +67,6 @@
 .general-info-item-content h3.anchor {
   margin-top: 2.5em;
   margin-bottom: 2px; }
-.general-info-item-content a.link-button {
```
Comment (bklukaczewski) [inline simple_ui/static/scss/content.css]: Shouldn't this file be removed from repo? Because it should be transpiled by SASS from .scss

Comment (reyrodrigues) [inline simple_ui/static/scss/content.css]: Is it? I think only the main.css is compiled on start. Content.css is needed so we can copy it to the app.

Comment (bklukaczewski) [inline simple_ui/static/scss/content.css]: there's a line in start.sh script:
`
sass simple_ui/static/scss/content.scss > simple_ui/static/scss/content.css; \
`
so it should get overwritten anyway.

**Comment by kkaczmarczyk** [comment]
let's merge it, as there is no harm, and it can be removed afterwards

**Pull request merged**